packages/runtime/src/templates/edge/middleware-runtime.ts

```diff
@@ -1,49 +1,13 @@
 import type { Context } from 'https://edge.netlify.com'
 // Available at build time
 import matchers from './matchers.json' assert { type: 'json' }
-import nextConfig from '../edge-shared/nextConfig.json' assert { type: 'json' }
 import edgeFunction from './bundle.js'
-import { buildResponse } from '../edge-shared/utils.ts'
+import { buildNextRequest, buildResponse } from '../edge-shared/utils.ts'
 import { getMiddlewareRouteMatcher, MiddlewareRouteMatch, searchParamsToUrlQuery } from '../edge-shared/next-utils.ts'
+import nextConfig from '../edge-shared/nextConfig.json' assert { type: 'json' }
 
 const matchesMiddleware: MiddlewareRouteMatch = getMiddlewareRouteMatcher(matchers || [])
 
-export interface FetchEventResult {
```

Comment:
Moved all of this into a shared file, so that we can have two separate entrypoints. They could go in a layer eventually.